### PR TITLE
New Russian translation (v3.8.1)

### PR DESCRIPTION
**Pull request by coeg701**
New russian translation has been made by our team of @realTiXoNyA @coeg701 @Yurikan4ik. Some fixes are possible.

### CODE REVIEW

**Comment by mty22** [comment]
Perfect! Thank you @coeg701 

Just need a native speaker to review your work.

Did you do all of this by yourself?

**Comment by coeg701** [comment]
No, we did it together with  @realTiXoNyA, @Yurikan4ik and me. We will get a reviewer soon, thanks.

**Comment by coeg701** [comment]
Also, it needs a few fixes too.

**Comment by mty22** [comment]
@coeg701 Any updates ? I'd love to get this one pushed out into the game. :) Just need that review~

File: ru.js [hunk 1]
```diff
@@ -0,0 +1,747 @@
+module.exports = `
+menu.btn.ranked Игра на рейтинг
+menu.btn.shop Магазин
```
Comment (ligmabots) [inline ru.js:3]: "magazin" is the correct word for shop in russian, but usually more for a physical grocery/supermarket

File: ru.js [hunk 2]
```diff
@@ -0,0 +1,747 @@
+module.exports = `
+menu.btn.ranked Игра на рейтинг
+menu.btn.shop Магазин
+menu.btn.social Сообщество
+menu.btn.maps Игры
```
Comment (ligmabots) [inline ru.js:5]: igrə - game

**Comment by mty22** [comment]
@coeg701 how are you doing with this? need anymore help with finding reviewers?

**Comment by coeg701** [comment]
@mty22 I mean, probably, yeah. I'm also not sure how does the reviewing process work. For now I have one I've talked to in my dms, but I think he's also not sure how this will work. I'll dm him again though.

**Comment by mty22** [comment]
> @mty22 I mean, probably, yeah. I'm also not sure how does the reviewing process work. For now I have one I've talked to in my dms, but I think he's also not sure how this will work. I'll dm him again though.

@coeg701  basically have him create a github account,  then get him to have a look here:

https://github.com/mty22/krunker.io-translations/pull/79/files

As other users have already done, have him add comments per line where there's issues by clicking the blue plus icon:

image

this will allow you to see, fix and click 'resolve' once we're good to go!

**Comment by coeg701** [comment]
@mty22 Now I get it, thanks. I will ask one to help with reviewing. Also I noticed some of lines in original en.js file are missing. When I load the translation file in game it shows "unlocalized string" in a few places.

**Comment by mty22** [comment]
OK, no problems. Lets forget the new `en.js` changes and get this translation done as it's been quite a while so far!

Let me know if you need help!

**Comment by coeg701** [comment]
Ok, no problems. Starting from tomorrow I will be finally completing this. I have already dm'ed two I have contact with about the review.

File: ru.js [hunk 3]
```diff
@@ -0,0 +1,777 @@
+module.exports = `
```
Comment (ligmabots) [inline ru.js:1]: I’ve checked over all the code, and all should be good now.

Comment (mty22) [reply ru.js:1]: perfect. thanks!

**Comment by coeg701** [comment]
Not sure about the reviewer, nothing was changed during the review basically (1% ig)
@realTiXoNyA - 30% (translation)
@coeg701 - 39% (translation/changes)
@Yurikan4ik - 30% (translation)
Reviewer - @ligmabots.

**Comment by coeg701** [comment]
IGN's and discord accounts:
realTiXoNyA - https://krunker.io/social.html?p=profile&q=%D0%9F%D0%B8%D0%BD%D0%B3%D0%B2%D0%B8%D0%BD
TiXoNyA
#5598
coeg701 - https://krunker.io/social.html?p=profile&q=coeg%E2%A0%80%E2%A0%80
coeg#0523
Yurikan4ik - https://krunker.io/social.html?p=profile&q=Yurikan4ik
Yurikan4ik
#7933

**Comment by coeg701** [comment]
Here's our IGN's and discord accounts:
realTiXoNyA - https://krunker.io/social.html?p=profile&q=%D0%9F%D0%B8%D0%BD%D0%B3%D0%B2%D0%B8%D0%BD
TiXoNyA#5598
coeg701 - https://krunker.io/social.html?p=profile&q=coeg%E2%A0%80%E2%A0%80
coeg#0523
Yurikan4ik - https://krunker.io/social.html?p=profile&q=Yurikan4ik
Yurikan4ik#7933

**Comment by coeg701** [comment]
Here's our IGNs and discord accounts:
@realTiXoNyA - https://krunker.io/social.html?p=profile&q=%D0%9F%D0%B8%D0%BD%D0%B3%D0%B2%D0%B8%D0%BD
`TiXoNyA#5598`
@coeg701 - https://krunker.io/social.html?p=profile&q=coeg%E2%A0%80%E2%A0%80
`coeg#0523`
@Yurikan4ik - https://krunker.io/social.html?p=profile&q=Yurikan4ik
`Yurikan4ik#7933`

**Comment by mty22** [comment]
perfect. ty.

**Comment by mty22** [comment]
roles sorted in KB.

**Comment by mty22** [comment]
@ligmabots can you send me your krunker ign and discord user please!

**Comment by ligmabots** [comment]
My IGN is the same as my username, ligmabots

**Pull request merged**